backend/apps/readings/analysis.py

```diff
@@ -3,9 +3,67 @@
 Analysis.py - analyses for dhmit/rereading wired into the webapp
 
 """
+from statistics import stdev
+from pathlib import Path
+
 from .models import StudentResponse
 
 
+def get_sentiments() -> dict:
+    """
+    Returns a dictionary of sentiment scores, with the keys being the word and the values being
+    their score
+
+    :return: dict mapping words to their sentiment scores
+    """
+    sentiment_path = Path('..', 'analysis', 'data', 'sentiments.txt')
```

Comment:
this path function assumes that the program is always being run from a particular directory.  It needs to begin with the `__file__` attribute.  See https://stackoverflow.com/questions/3430372/how-do-i-get-the-full-path-of-the-current-files-directory -- the answer beginning "Using Path is the recommended way since Python 3"

Reply:
also: I always put some path constants in config files -- check out `config.settings.base`, import `BACKEND_DIR` or `PROJECT_ROOT` from there, and build paths relative to those. They'll stay stable over the course of the project, whereas this file might move.